content/tutorials/quickstart/batch-minting.md

```diff
@@ -0,0 +1,494 @@
+  results += "Tickets generated, minting NFTokens.\n\n"
+  document.getElementById('standbyResultField').value = results
+```
+
+Use a `for` loop to create the NFTokens one at a time, up to the NFTokenCount you specified.
+
+```javascript
+  for (let i=0; i < nftokenCount; i++) {
+		const transactionBlob = {
+			"TransactionType": "NFTokenMint",
+			"Account": standby_wallet.classicAddress,
+			"URI": xrpl.convertStringToHex(standbyTokenUrlField.value),
+			"Flags": parseInt(standbyFlagsField.value),
+			"TransferFee": parseInt(standbyTransferFeeField.value),
+			"Sequence": 0,
+```
+
+Use the loop variable to step through the array. Set the `LastLedgerSequence` to `null`.
+
+```javascript
+			"TicketSequence": tickets[i],
+			"LastLedgerSequence": null,
+			"NFTokenTaxon": 0 
+		}
+```
+
+Submit the signed transaction hash.
+
+```javascript
+		const tx =  client.submit(transactionBlob, { wallet: standby_wallet} )
+  }
+```
+
+Use the same logic as `getBatchNFTokens`, above, to get the list of current NFTokens.
+  
+```javascript
+  results += "\n\nNFTs:\n"
+  let nfts = await client.request({
+    method: "account_nfts",
+    account: standby_wallet.classicAddress,
+    limit: 400
+  })
+
+  results += JSON.stringify(nfts,null,2)
+  while (nfts.result.marker != null)
+  {
+		nfts = await client.request({
+			method: "account_nfts",
+			account: standby_wallet.classicAddress,
+			limit: 400,
+			marker: nfts.result.marker
+		})
+    results += '\n' + JSON.stringify(nfts,null,2)
+  }
+```
+
+Report the results.
+
+```javascript
+	results += '\n\nTransaction result: '+ tx.result.meta.TransactionResult
+	results += '\n\nnftokens: ' + JSON.stringify(nfts, null, 2)
+	document.getElementById('standbyBalanceField').value = 
+		(await client.getXrpBalance(standby_wallet.address))
+	document.getElementById('standbyResultField').value = results
+```
+
+Disconnect from the XRP Ledger.
+
+```javascript
+  client.disconnect()
+} // End of batchMint()
+```
+
+## 7.batch-minting.html
+
+For this form:
+* The unnecessary script references are removed. 
+* The unused fields and buttons are removed.
+* The `standbyResultField` capacity is set to the maximum value, 524288.
+
+```html
+<html>
+  <head>
+    <title>Token Test Harness</title>
+    <link href='https://fonts.googleapis.com/css?family=Work Sans' rel='stylesheet'>
+    <style>
+       body{font-family: "Work Sans", sans-serif;padding: 20px;background: #fafafa;}
+       h1{font-weight: bold;}
+       input, button {padding: 6px;margin-bottom: 8px;}
+       button{font-weight: bold;font-family: "Work Sans", sans-serif;}
+       td{vertical-align: middle;}
+    </style>
+    <script src='https://unpkg.com/xrpl@2.2.3'></script>
+    <script src='ripplex1-send-xrp.js'></script>
+    <script src='ripplex3-mint-nfts.js'></script>
+    <script src='ripplex7-batch-minting.js'></script>
+    <script>
+      if (typeof module !== "undefined") {
+        const xrpl = require('xrpl')
+      }
+     
+    </script>
```

Comment:
I think this can be omitted entirely for this example code.

The `.js` files I use as examples in some other places do this sort of thing so that they can either be included in an HTML page, or run directly with Node.js; it doesn't make sense to include this code inline in an HTML file.

Reply:
Will catch this next time around.